packages/blocks/src/components/format-quick-bar/icons.ts

```diff
@@ -288,3 +288,10 @@ export const ArrowDownIcon = svg`<svg
     d="M9.97678 11.318C9.96662 11.3111 9.94678 11.2967 9.9152 11.2699C9.82898 11.1967 9.71943 11.0881 9.52859 10.8972L6.39901 7.76763C6.08659 7.45521 5.58006 7.45521 5.26764 7.76763C4.95522 8.08005 4.95522 8.58658 5.26764 8.899L8.39722 12.0286C8.40353 12.0349 8.40982 12.0412 8.41611 12.0475C8.5813 12.2127 8.73751 12.3689 8.87972 12.4897C9.03447 12.621 9.22442 12.7566 9.46951 12.8363C9.81429 12.9483 10.1857 12.9483 10.5305 12.8363C10.7756 12.7566 10.9655 12.621 11.1203 12.4897C11.2625 12.3689 11.4187 12.2127 11.5839 12.0475L14.7323 8.899C15.0448 8.58658 15.0448 8.08005 14.7323 7.76763C14.4199 7.45521 13.9134 7.45521 13.601 7.76763L10.4714 10.8972C10.2806 11.0881 10.171 11.1967 10.0848 11.2699C10.0532 11.2967 10.0334 11.3111 10.0232 11.318C10.0079 11.3211 9.9921 11.3211 9.97678 11.318Z"
   />
 </svg>`;
+
+export const CodeBlockIcon = svg`
```

Comment:
The code icon already exists in icons.ts

https://github.com/toeverything/blocksuite/blob/14e22a29412f583d0a6e2582f22b91d8df13897f/packages/blocks/src/components/format-quick-bar/icons.ts#L135-L151

Reply:
> <img alt="image" width="285" src="https://user-images.githubusercontent.com/14026360/209392706-c8dfb22c-1ffb-4ce1-bd6e-8132b1a534de.png">
> 
> <img alt="image" width="369" src="https://user-images.githubusercontent.com/14026360/209392753-63fbe434-7316-4148-93d6-39da1392c2dd.png">
> 
> css is incorrect.

Thanks! It was caused by wrong z-index, and has been fixed